code/go/internal/validator/test/packages/good/data_stream/ilm_policy_hidden/manifest.yml

```diff
@@ -0,0 +1,7 @@
+title: Hidden datastream and ilm policy overrride
```

Comment:
The correct spelling is `data stream` not `datastream`. Please adjust the entire PR.